index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+intentionally mirror settable attributes and methods of the
+[HtmlMediaElement](https://html.spec.whatwg.org/multipage/media.html#htmlmediaelement).
+
+: source
+:: Change the [=media resource=] URL. See
+    [HtmlMediaElement.src](https://html.spec.whatwg.org/multipage/media.html#dom-media-src)
+    for more details. Must not be used in the initial controls of the
+    remote-playback-start-request message (which already contains a list of URLs).
+ 
+: preload
+:: Set how aggresively to preload media. See
+    [HtmlMediaElement.preload](https://html.spec.whatwg.org/multipage/media.html#dom-media-preload)
+    for more details. Should only be used in the initial controls of the
+    remote-playback-start-request message or when the source is changed.  If not
+    set in the initial controls, it is left to the receiver to decide.  This is
+    optional for the receiver to support and if not supported, the receiver will
+    behave as though it were never set.
+
+: loop
+:: Set whether or not to loop media. See
+    [HtmlMediaElement.loop](https://html.spec.whatwg.org/multipage/media.html#dom-media-loop)
+    for more details. Should only be used in the initial control of the
+    remote-playback-start-request.  If not set in the initial controls, it is
+    assumed to be false.
+
+: paused 
+:: If true, pause; if false, resume.  See
+    [HtmlMediaElement.pause()](https://html.spec.whatwg.org/multipage/media.html#dom-media-pause).
+    and
+    [HtmlMediaElement.play()](https://html.spec.whatwg.org/multipage/media.html#dom-media-play)
+    for more details.  If not set in the initial controls, it is left to the
+    receiver to decide.
+
+: muted
+:: If true, mute; if false, unmute.  See
+    [HtmlMediaElement.muted](https://html.spec.whatwg.org/multipage/media.html#dom-media-muted)
+    for more details.  If not set in the initial controls, it is left to the
+    receiver to decide.
+
+: volume
+:: Set the audio volume in the range from 0.0 to 1.0 inclusive.  See
+    [HtmlMediaElement.volume](https://html.spec.whatwg.org/multipage/media.html#dom-media-volume)
+    for more details.  If not set in the initial controls, it is left to the
+    receiver to decide.
+
+: seek
+:: Seek to a precise time.  See
+    [HtmlMediaElement.currentTime](https://html.spec.whatwg.org/multipage/media.html#dom-media-currenttime)
+    for more details. 
+
+: fast-seek
+:: Seek to an approximate time as fast as possible. See
+    [HtmlMediaElement.fastSeek()](https://html.spec.whatwg.org/multipage/media.html#dom-media-fastseek)
+    for more details.
+
+: playback-rate
+:: Set the rate a which the media plays.  See
+    [HtmlMediaElement.playbackRate](https://html.spec.whatwg.org/multipage/media.html#dom-media-playbackrate)
+    for more details.  If not set in the initial controls, it is left to the
+    receiver to decide.  This is optional for the receiver to support and if not
+    supported, the receiver will behave as though it were never set.
```

Comment:
If the receiver doesn't support playback rate, that means that control message to set it to 1.5 would be ignored by the receiver.  Which means that no state event would be sent, because the receiver's value doesn't change.  So the controller would think the playback rate is 1.5 and the receiver is actually using 1.0?

In response to a control message, we could have the receiver always send a event for any state member that could have changed (even if it didn't), to make sure the controller stays in sync.

Reply:
There's a remote-playback-modify-response with a state, so the controller would see the playback rate didn't change.